operatorapi/volumes.go

```diff
@@ -218,3 +228,36 @@ func getPVCEventsResponse(session *models.Principal, params operator_api.GetPVCE
 	})
 	return retval, nil
 }
+
+func getTenantCSResponse(session *models.Principal, params operator_api.ListTenantCertificateSigningRequestParams) (*models.CsrElement, *models.Error) {
+	ctx, cancel := context.WithCancel(context.Background())
```

Comment:
```suggestion
	ctx, cancel := context.WithCancel(params.HTTPRequest.Context())
```

For now on, every time we need to use context on a Response function lets grab it from the HTTPRequest object in params, in contains important context information helpful for logging errors

Reply:
Got it, didn't know @Alevsk thank you for the review 👍